addOns/requester/src/main/java/org/zaproxy/zap/extension/requester/RequesterNumberedTabbedPane.java

```diff
@@ -30,7 +30,7 @@ public RequesterNumberedTabbedPane() {
     @Override
     public void addDefaultTab() {
         ManualHttpRequestEditorPanel requestPane =
-                new ManualHttpRequestEditorPanel(true, "requesterpanel");
+                new ManualHttpRequestEditorPanel(true, RequesterPanel.PANEL_NAME);
```

Comment:
If the config key is the same everywhere this should not be parameter (applies to both parameters). We should remove `PANEL_NAME` that's not used anywhere.